en/components/chip.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Chip Component - Native Angular | Ignite UI for Angular
+_description: The Ignite UI for Angular Chip component provide compact elements that represent an input, attribute, or action.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Native Angular Components, Chip, Chip Component, ChipArea, ChipArea Component
+---
+
+### Chip
+
+`igxChip` Chips are compact components that represent discrete information.
+
+#### Demo
+
+<div class="sample-container loading" style="height:550px">
+    <iframe id="chip-iframe" src='{environment:demosBaseUrl}/chip-sample' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="chip-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+#### Initializing Chips
+
+The `IgxChipComponent` is the main class for a chip elemenent and the `IgxChipsAreaComponent` is the main class for the chip area, it is used to manage the chips defined inside it. It is used when handling more complex scenarios that require interaction between chips (dragging, selection, navigation and etc.).The IgxChipComponent requires an id to be defined so that the different chips can be easily distinguished.
+
+Example of using `igxChip` with `igxChipArea`:
+
+```html
+<igx-chips-area>
+    <igx-chip *ngFor="let chip of chipList" [id]="chip.id">
+        <label igxLabel>{{chip.text}}</label>
+    </igx-chip>
+</igx-chips-area>
+```
+
+### Features
+
+#### Selection
+
+Selection is disabled by default, but can be enabled with an option called `selectable`. The selecting is done by focusing the chip either by clicking of using the `Tab` button and the pressing `Space`. An event is fired when the selection state of the `igxChip` changes. If a chip is already selected it can be deselected by pressing the `Space` button again while the chip is focused.
```

Comment:
Selection can be done by clicking now. The `The selecting is done by focusing the chip either by clicking of using the `Tab` button and the pressing `Space`. sentence would need to be reworded to include that information. Both Mouse Click and Space are viable options for selection.